packages/sales-channel/src/models/sales-channel.ts

```diff
@@ -0,0 +1,50 @@
+import { generateEntityId } from "@medusajs/utils"
```

Comment:
comment: If we choose to apply these schema changes, there are likely some other places that need updating too.